contrib/libcityhash/include/city.h

```diff
@@ -52,6 +52,10 @@ typedef uint32_t uint32;
 typedef uint64_t uint64;
 typedef std::pair<uint64, uint64> uint128;
 
+/// This is a version of CityHash that predates v1.0.3 algorithm change.
```

Comment:
Almost Ok, but this is not the motivation (why we need this version).
I will write by myself, if you want.